packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -32,6 +35,24 @@ describe('<DataGrid />', () => {
 
     // Adapation of describeConformance()
     describe('Material-UI component API', () => {
+      it(`resizable true should throw error`, () => {
+        defaultProps.columns[0].resizable = true;
+        expect(() =>
+          render(
+            <div style={{ width: 300, height: 300 }}>
+              <DataGrid {...defaultProps} />
+            </div>,
+          ),
+        ).to.throw(
+          [
+            `Material-UI: \` column.resizable = true \` is not a valid prop.`,
+            'Column resizing is not available in the MIT version',
```

Comment:
I believe `to.throw` accepts a partial match. I think only matching with the first sentence will be enough to make sure the error isn't coming from somewhere else, and at the same time, make the test easier to write and less likely to fail the next time we fix a typo or slightly improve the error message.